### ISSUE TITLE
Option to use the phone's brightness

### ISSUE THREAD

**Issue by lfrankel** [ram-on/SkyTube#812]
While playing, SkyTube seems to always want to set the brightness to its own special internal value, that can be made brighter or darker with a swipe.

For me, this is a major pain in the backside. I just want the screen to stay whatever brightness the phone already had it as.

I use https://twilight.urbandroid.org/ to make the screen darker after sundown. Then I open up a video in SkyTube and it blasts my eyeballs out, and I have to lower the brightness manually. Then the next day, if I want to watch something on SkyTube I have to raise it back up because now it's way too dark.

Besides the fact that my phone already has as a brightness slider.

Can we please have an option to just not mess with the brightness at all? Please?

**Comment by gzsombor** [comment]
This could be an easy feature, for someone - you would need to modify YouTubePlayerV2Fragment.VideoBrightness constructor, to avoid setting the brightness at startup, somewhere around here: https://github.com/ram-on/SkyTube/blob/d99768d2766ff81e8dc4ff1ca9d28cfd84754ffe/app/src/main/java/free/rm/skytube/gui/fragments/YouTubePlayerV2Fragment.java#L984
And add a checkbox to one of the Preference activity, to actually set it.

**Comment by BugReportering** [comment]
Not messing with brightness in the app at all would solve the issue I was having too, reported here:  
https://github.com/SkyTubeTeam/SkyTube/issues/932
The app should respect the phones set brightness, not try to re-implement the wheel.

**Comment by BugReportering** [comment]
If you followed this advice with the audio as well, that app should respect the media volume of the phone, not implement it's own separate audio volume rules, it would also solve this issue I reported: https://github.com/SkyTubeTeam/SkyTube/issues/931

**Comment by gzsombor** [comment]
Adjusting the brightness, and the volume easily from the playback is an important feature. But if you have better ideas, please send patches

**Comment by drogga** [comment]
...yeah..., both gestures (features) are nice to have, they are just implemented badly, NewPipe handles them better, also BSPlayer has options in its preferences to respect them or not, so something similar can be used here too - maybe...
...if not, then splitting the "disable video gestures" option to 2 or 3, so at least only the brightness one can only be disabled would be kind\sort of a quick fix or workaround for the time being.

**Comment by BugReportering** [comment]
Ifrankel, myself, and now even drogga have all  given you better ideas.  You want us to write the code for you also?

**Comment by gzsombor** [comment]
No, I don't want to write code for me. I suggested you to write code for yourself. I'm already wrote lot's of code for you :wink:

**Comment by BugReportering** [comment]
You didn't write any code for me, as I am unable to use your app due to the bugs I've already reported, most importantly the hard lockup. I still need to use NewPipe Legacy or IceCatMobile for my YouTubing.

**Comment by gzsombor** [comment]
Couple of mistakes: 
* If you ever used the app in the last 3 years, it contained code written by me, so I wrote code for you.  
* It's not **my** app, it's an **open source** app, anyone can contribute to it, and improve it. 
* This is not my job or duty to work for you in my free time, here I've offered you - or any other people who thinks this is an important feature - suggestions, where and how can it be implemented.
* Feel free to use any other app, or using an older version of the app, maybe 2.69, which doesn't contain any code written by me :smiley:

**Comment by BugReportering** [comment]
> Couple of mistakes:
> 
>     * If you ever used the app in the last 3 years, it contained code written by me, so I wrote code for you.
> 
>     * It's not **my** app, it's an **open source** app, anyone can contribute to it, and improve it.
> 
>     * This is not my job or duty to work for you in my free time, here I've offered you - or any other people who thinks this is an important feature - suggestions, where and how can it be implemented.
> 
>     * Feel free to use any other app, or using an older version of the app, maybe 2.69, which doesn't contain any code written by me smiley

1: I never even tried to use this app before yesterday. So, again, you didn't write any code for me.
2: If it's not your app why do you seem to be the one taking charge of it, and taking so much offense to the bug reports?
3: I already do use another app, as I have previously stated.
4: Your smug attitude only helps to further push people away from your apps, or any code written by you, which might I add is very poor quality code anyways.

**Comment by gzsombor** [comment]
Ok, so you opened 3 2-liner bugs even before you tried the app, and now you even insulted the people in #881, who actually put effort to improve the app. So please chill down, and start sending patches, if you know, how to write good quality code.

**Comment by drogga** [comment]
@BugReportering Calm down please, take it easy, gzsombor is not your enemy.
2: - Because there's no one else..., nobody seems to care and actually contribute with code, only ideas, but the problem is that nobody is helping gzsombor, he's currently the only one doing something, without him the project will be completely dead.
@gzsombor - You have my support, keep up the work when you can, and let's hope that as much as possible suggestions can be implemented in the next months (of this year) ;)

**Comment by BugReportering** [comment]
> @BugReportering Calm down please, take it easy, gzsombor is not your enemy.
> 2: - Because there's no one else..., nobody seems to care and actually contribute with code, only ideas, but the problem is that nobody is helping gzsombor, he's currently the only one doing something, without him the project will be completely dead.
> @gzsombor - You have my support, keep up the work when you can, and let's hope that as much as possible suggestions can be implemented in the next months (of this year) ;)

As well, I am not his enemy, but he is being disrespectful and treating me as such, because I reported bugs in this app, which apparently and according to him, is not even his.

If you are going to contribute code to an open source project and have it public on github, then be tough enough to receive constructive criticism and feedback.  You don't have to agree with it, but you also don't have to turn into a crying child about it either.

**Comment by gzsombor** [comment]
> > @BugReportering Calm down please, take it easy, gzsombor is not your enemy.
> > 2: - Because there's no one else..., nobody seems to care and actually contribute with code, only ideas, but the problem is that nobody is helping gzsombor, he's currently the only one doing something, without him the project will be completely dead.
> > @gzsombor - You have my support, keep up the work when you can, and let's hope that as much as possible suggestions can be implemented in the next months (of this year) ;)
> 
> As well, I am not his enemy, but he is being disrespectful and treating me as such, because I reported bugs in this app, which apparently and according to him, is not even his.
> 
> If you are going to contribute code to an open source project and have it public on github, then be tough enough to receive constructive criticism and feedback. You don't have to agree with it, but you also don't have to turn into a crying child about it either.

Pardon me, where were I disrespectful to you? The only interaction I had with you, is when I pointed out, that if - instead of filling the bug report template - you only wrote one line of pretty generic problem, that it's impossible to know, what was happened, and how. Or if it's already fixed, whatever. 
 Sure, you can criticize my code, if you find problems in it, but if you criticize me for a code, what someone else wrote it, maybe long time before I even knew about this project, I will find you very confused and offensive. 
 And if you are going to use an open source project, then you should be prepared, that the project is driven by code contributions, by people in their free time. You don't have to agree with their priorities, but if someone has other priorities than to fix your bug or implement your proposed feature, then you should not start insulting them, that their 'programming is so poor',etc... This is not what anyone would call as a 'constructive criticism'.

**Comment by drogga** [comment]
@gzsombor I've commented on 2 PRs (1 Open & 1 Closed), so please at least react with a emoji if \ when you read them (in case \ when you don't want to reply for some reason), so I'll at least know that you saw them... ;)
Thanks.

**Comment by gzsombor** [comment]
@drogga : I could only found 1 recent comment of you, which I haven't reacted.

**Comment by drogga** [comment]
Oh, it seems that search a comment by username \ nickname in Pull Requests doesn't actually works. @GitHub - Please fix it.

I meant:

["Dark theme" - {Open}] https://github.com/SkyTubeTeam/SkyTube/pull/880
["Fix Images in the README" - {Closed}] https://github.com/SkyTubeTeam/SkyTube/pull/929
